src/index.d.ts

```diff
@@ -69,6 +69,7 @@ export type DropzoneAreaBaseProps = {
   onDelete?: (deletedFileObject: FileObject, index: number) => void;
   onDrop?: (files: File[], event: DropEvent) => void;
   onDropRejected?: (files: File[], event: DropEvent) => void;
+  onAlert?: (message: string, variant: string) => void;
```

Comment:
Since `variant` possible values are know instead of `string` we can use the existing `AlertType` type.